spec/models/food_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Food, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
- [ ] [REQUIRED] Hi team, kindly ensure you write proper units and integration tests for this project as stated in the project requirement.